packages/config/config/devices/0x014a/tstat-tbz500.json

```diff
@@ -0,0 +1,712 @@
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 3,
+			"maxValue": 8,
+			"defaultValue": 3
+		},
+		{
+			"#": "16",
+			"label": "Aux Heat Off Threshold",
+			"description": "Sets the delta from setpoint that stage 3 heating stops. Stage 3 turns off at setpoint + Delta Stage 3.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 7,
+			"defaultValue": 0
+		},
+		{
+			"#": "17",
+			"label": "Cooling Stage 1 On Threshold",
+			"description": "Sets the delta from setpoint that stage 1 cooling starts.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 7,
+			"defaultValue": 1
+		},
+		{
+			"#": "18",
+			"label": "Cooling Stage 1 Off Threshold",
+			"description": "Sets the delta from setpoint that stage 1 cooling stops. Stage 1 turns off at setpoint - Delta Stage 1.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 0
+		},
+		{
+			"#": "19",
+			"label": "Cooling Stage 2 On Threshold",
+			"description": "Sets the delta from setpoint that stage 2 cooling starts.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 2,
+			"maxValue": 8,
+			"defaultValue": 2
+		},
+		{
+			"#": "20",
+			"label": "Cooling Stage 1 Off Threshold",
+			"description": "Sets the delta from setpoint that stage 2 cooling stops. Stage 2 turns off at setpoint - Delta Stage 2.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 7,
+			"defaultValue": 0
+		},
+		{
+			"#": "21",
+			"label": "Mechanical Status",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 256,
+			"unsigned": true,
+			"readOnly": true,
+			"options": [
+				{
+					"label": "MECH_H1",
+					"value": 1
+				},
+				{
+					"label": "MECH_H2",
+					"value": 2
+				},
+				{
+					"label": "MECH_H3",
+					"value": 4
+				},
+				{
+					"label": "MECH_C1",
+					"value": 8
+				},
+				{
+					"label": "MECH_C2",
+					"value": 16
+				},
+				{
+					"label": "PHANTOM_F",
+					"value": 32
+				},
+				{
+					"label": "MECH_F",
+					"value": 64
+				},
+				{
+					"label": "MANUAL_F",
+					"value": 128
+				},
+				{
+					"label": "Reserved",
+					"value": 256
+				}
+			]
+		},
+		{
+			"#": "22",
+			"label": "SCP Status", // SCP is not defined in the manual. Perhaps State Condition Parameter.
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 128,
+			"unsigned": true,
+			"readOnly": true,
+			"options": [
+				{
+					"label": "STATE_HEAT",
+					"value": 1
+				},
+				{
+					"label": "STATE_COOL2",
+					"value": 2
+				},
+				{
+					"label": "STATE_2ND",
+					"value": 4
+				},
+				{
+					"label": "STATE_3RD",
+					"value": 8
+				},
+				{
+					"label": "STATE_FAN",
+					"value": 16
+				},
+				{
+					"label": "STATE_LAST",
+					"value": 32
+				},
+				{
+					"label": "STATE_MOT",
+					"value": 64
+				},
+				{
+					"label": "STATE_MRT",
+					"value": 128
+				}
+			]
```

Comment:
These also look like they are supposed to be partials, but again I'm not sure if that makes sense here.
Are you able to try if there is ever a value other than the listed ones?

Reply:
This one, unlike No. 23, which is clearly meant to be a enable mask for various autosends, appears to be like the previous: just a byte that can contain a value that represents a particular STATE. All of these are mutually exclusive, with the possible exception of STATE_LAST. I don't know what it represents. Both 21 and 22 are triggered by setting the two MSBs in No. 23, so we know these values are reported. Again, I didn't think these fit the partials profile.

Reply:
fine with me 👍🏻